CHANGELOG.md

```diff
@@ -13,9 +13,12 @@ Releases
 
 ### 1.8.0
 
+* [-] on Mac OSX and FreeBSD correctly use interface name for routing
+* [~] on Mac OSX and FreeBSD moved netstat parsing output to higher debug level
+* [~] when logging traffic also show http traffic (not only tunneled traffic)
 * [~] improved error message in case of login failure
 * [~] Require root privileges for running. They are needed at various places. 
-      previously, just a warning was issued, but in later stage things have failed. 
+      Peviously, just a warning was issued, but in later stage things have failed.
```

Comment:
small typo here

Reply:
Thanks!

I was more talking about commit titles than changelog entries, but anyway this is fine :+1: